source/Playnite/Localization/LocSource.xaml

```diff
@@ -936,6 +936,7 @@ If you want to install or uninstall additional integrations, use "Add-ons" optio
     <sys:String x:Key="LOCTopPanelMainButtonMove">Move main menu button to the sidebar</sys:String>
     <sys:String x:Key="LOCTopPanelExplorerSwitch">Explorer panel</sys:String>
     <sys:String x:Key="LOCTopPanelSelectRandomGameButton">Random game picker</sys:String>
+    <sys:String x:Key="LOCTopPanelViewSelectRandomGameButton">View random game selector</sys:String>

```

Comment:
Not sure about this description. It in my opinion sounds like it will open random game selector, especially if you see it as a tooltip on top panel button.

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3032/commits/4ea1b34d63b8e95f02245f732806ab4a1861bbf2